### PR TITLE
Modifications for raw3-3 with Frida and fixes inside light_controller

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

File: cob_bringup/components/light.launch [hunk 1]
```diff
@@ -3,15 +3,9 @@
 
 	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
-	<arg name="sim" default="false" />
```
Comment (NadiaHG) [inline cob_bringup/components/light.launch:6]: why did you delete cob_light_sim?

the simulated cob_light node should publish the marker topic but not communicate to the hardware device

Comment (benmaidel) [inline cob_bringup/components/light.launch:6]: Since there is no cob_light_sim node, we do not need this launch configuration anymore. The new cob_light package enables simulation mode automatically if no hardware is present. Publishing the LightMarker can be enabled by setting a Flag inside light.yaml configuration.
